TRACKER_ROOMBA_IDS: [1, 4, 6, 7]